lib/network/modules/InteractionHandler.js

```diff
@@ -77,21 +77,21 @@ class InteractionHandler {
    */
   setOptions(options) {
     if (options !== undefined) {
-      // extend all but the values in fields
-      let fields = ['hideEdgesOnDrag', 'hideEdgesOnZoom', 'hideNodesOnDrag','keyboard','multiselect','selectable','selectConnectedEdges'];
-      util.selectiveNotDeepExtend(fields, this.options, options);
-
       // remove zoomMin and/or zoomMax if they aren't numbers
-      let { zoomMin, zoomMax } = this.options;
+      const { zoomMin, zoomMax } = options;
 
-      if (typeof zoomMin !== undefined && typeof zoomMin !== 'number') {
-        delete this.options.zoomMin;
+      if (!util.isNumber(zoomMin) || Number.isNaN(zoomMin)) {
+        delete options.zoomMin;
       }
 
-      if (typeof zoomMax !== undefined && typeof zoomMax !== 'number') {
-        delete this.options.zoomMax;
+      if (!util.isNumber(zoomMax) || Number.isNaN(zoomMax)) {
+        delete options.zoomMax;
```

Comment:
What if the maximum is smaller than the minimum? What if someone inputs negative number? These situations don't seem okay to me.